services/web/client/source/class/qxapp/data/Store.js

```diff
@@ -50,7 +50,8 @@ qx.Class.define("qxapp.data.Store", {
     getRole: function() {
       // Temporary HACK
       const userEmail = qxapp.auth.Data.getInstance().getEmail();
-      if (userEmail.includes("itis.swiss") || userEmail.includes("oetiker.ch")) {
+      const inHouse = ["itis.swiss", "speag.swiss", "oetiker.ch"];
+      if (inHouse.some(v => userEmail.indexOf(v) >= 0)) {
```

Comment:
your soooo old style :)

how abot using and object/hash or regexp maching for this kind of test ?